src/config/alchemy-sdk.ts

```diff
@@ -11,5 +11,11 @@ const polygonNetworkSettings = {
   network: Network.MATIC_MAINNET,
 }
 
+const pTokenNetworkSettings = {
+  apikey: config.alchemyApiKey,
+  network: Network.ETH_MAINNET,
+}
+
 export const ethAlchemy = new Alchemy(ethNetworkSettings)
 export const polygonAlchemy = new Alchemy(polygonNetworkSettings)
+export const ptokenAlchemy = new Alchemy(pTokenNetworkSettings)
```

Comment:
You dont need to create another instance for ptoken. eth instance works for it

Reply:
Ha! I'll fix it.

Reply:
this is still there @Adebesin-Cell ...remove the ptokenAlchemy